TemplateApplication/Contacts/Contacts.swift

```diff
@@ -12,7 +12,13 @@ import SpeziContact
 import SwiftUI
 
 
-/// Displays the contacts for the Spezi Template Application.
+/// Display contacts for your app.
+///
+/// Modify the contact details for your app in the Contacts view.
+///
+/// [Spezi Contact GitHub](https://github.com/StanfordSpezi/SpeziContact)
```

Comment:
Maybe something like here and in other parts?
```suggestion
/// You can learn more about [Spezi Contact in the GitHub repository](https://github.com/StanfordSpezi/SpeziContact)
```